test_launch_ros/.eggs/README.txt

```diff
@@ -0,0 +1,6 @@
+This directory contains eggs that were downloaded by setuptools to build, test, and run plug-ins.
```

Comment:
@jacobperron hmm do we need to keep `.eggs` around?

Reply:
Oops, removed (a836234)